### DISCUSSION TITLE
I want to declare a large area of blocks for the server or individual players, can it be operated with administrator instructions? What is the command, please? I didn't understand the wiki comments

### DISCUSSION THREAD

**Discussion by Luoboawa** [TownyAdvanced/Towny#7558] [Q&A]
I want to declare a large area of blocks for the server or individual players, can it be operated with administrator instructions? What is the command, please? I didn't understand the wiki comments

**Comment by LlmDl** [comment]
It sounds like you want an area where Towny isn't used.

Consider using worldguard-towny, to make a space where towns cannot be made. Then use worldguard or another claiming plugin there.

Or use a whole different world, with towny's `claimable` world setting disabled there.
